editors/vscode/README.md

```diff
@@ -27,13 +27,13 @@ Configure Rome as the default formatter for supported files to ensure that VS Co
 You can also enable Rome for specific languages only:
 
 - [Open the `settings.json`](https://code.visualstudio.com/docs/getstarted/settings#_settingsjson): open the _Command Palette_(<kbd>Ctrl</kbd>/<kbd title="Cmd">⌘</kbd>+<kbd title="Shift">⇧</kbd>+<kbd>P</kbd>) and select _Preferences: Open User Settings (JSON)_
-- And set the `editor.defaultFormatter` to `biome.rome` for the desired language
+- And set the `editor.defaultFormatter` to `biomejs.biome` for the desired language
```

Comment:
Is this correct?

Reply:
The publisher is named `biomejs`. We could publish a package named `biome`?

Reply:
Oh no, I am entirely on board with it! Consistency FTW